docs/eventing/flows/parallel.md

```diff
@@ -42,4 +42,4 @@ Parallel has three parts for the Status:
 
 ## Examples
```

Comment:
Add an alias to the frontmatter of this page, so that it will automatically redirect from the old URL to this new location:
```
---
title: "Parallel"
weight: 20
type: "docs"
aliases:
   - /docs/eventing/parallel.md
---
```